doc/source/io.rst

```diff
@@ -168,21 +168,24 @@ true_values : list, default ``None``
   Values to consider as ``True``.
 false_values : list, default ``None``
   Values to consider as ``False``.
-skipinitialspace : boolean, default ``False``
+skip_initial_space : boolean, default ``False``
   Skip spaces after delimiter.
-skiprows : list-like or integer, default ``None``
+skip_rows : list-like or integer, default ``None``
   Line numbers to skip (0-indexed) or number of lines to skip (int) at the start
   of the file.
-skipfooter : int, default ``0``
+skip_footer : int, default ``0``
   Number of lines at bottom of file to skip (unsupported with engine='c').
+skipfooter : int, default ``None``
+  DEPRECATED: this argument will be removed in a future version. Please
+  use the ``skip_footer`` parameter instead.
```

Comment:
Personally I don't think that is a reason to do differentely for this one (in the end, for all the others, the non-underscore versions were *also* in the signature)